src/Language/PureScript/Errors.hs

```diff
@@ -535,25 +539,25 @@ prettyPrintSingleError full level e = prettyPrintErrorMessage . positionHintsFir
                                           , indent $ prettyPrintValue expr
                                           ]) binding
     renderSimpleErrorMessage (TypesDoNotUnify t1 t2)
-      = paras [ line "Cannot unify type"
+      = paras [ line "The type"
               , indent $ typeAsBox t1
-              , line "with type"
+              , line "cannot be made equal to the type"
```

Comment:
I'm really not sure about that wording ("equal")! I guess it's not wrong, as that's what the process of applying the substitution returned by unification is, but it sounds weird to me...

The biggest problem people seem to have with this error message is they don't know which way the types go: is `t1` the type we have and `t2` the type it should be? Or vice versa. I think Haskell does a pretty good job of making it clear:

```
Couldn't match expected type X with actual type Y
```